fflas-ffpack/ffpack/ffpack_sss.inl

```diff
@@ -259,105 +259,296 @@ namespace FFPACK{
+                    {
+                        //              std::cout << "brow = "<<brow << std::endl;
+                        //std::cout << "N - sbrow+2  = "<<N - s * (brow + 2) << std::endl;
+                        /*FFLAS::WriteMatrix(std::cout<<"Before PLUQ, H = "<<std::endl, Fi, 2 * s,
+                          N - s* (brow + 2), H + N * s * brow + s * (brow + 2),
+                          N);*/
+                        r = FFPACK::PLUQ (Fi, FFLAS::FflasNonUnit, s*(2),
+                                          N - s*(brow + 2), H + N * s * brow + s * (brow + 2), N,
+                                          p, q);
+                        // FFLAS::WriteMatrix(std::cout<<"After PLUQ, H = "<<std::endl, Fi, 2 * s,
+                        //                 N - s* (brow + 2), H + N * s * brow + s * (brow + 2),
+                        //                 N);
+                        // pL -> [W_{brow + 2} \\ U_{brow + 2}]
+                        // 1) L -> Temp
+                        FFLAS::fzero (Fi, 2 * s, s, Temp, s);           
+                        FFPACK::getTriangular(Fi, FFLAS::FflasLower, FFLAS::FflasUnit, 2*s,
+                                              N - s * (brow + 2), r,
+                                              H + s * N * brow + s * (brow + 2),
+                                              N, Temp, s, true);
+                        // 2) p * Temp -> Temp
+                        FFPACK::applyP (Fi, FFLAS::FflasLeft, FFLAS::FflasNoTrans, s,
+                                        0, 2*s - 1, Temp, s, p);
+                        // 3) Temp1 -> W_{brow + 2}
+                        FFLAS::fassign (Fi, s, s, Temp, s, W + ldw * s * brow, ldw);
+                        // 4) Temp2 -> U_{brow + 2}
+                        FFLAS::fassign (Fi, s, s, Temp + s * s, s, U + ldu * s * (brow + 1), ldu);
+                        // Uq -> [V_{brow + 3} & H]
+                        FFPACK::getTriangular(Fi, FFLAS::FflasUpper, FFLAS::FflasNonUnit, s, N - s * (brow + 2), r,
+                                              H + s * N * brow + s * (brow + 2), N,
+                                              H + s * N * (brow + 1) + s * (brow + 2), N); // Remove L
+                        FFPACK::applyP (Fi, FFLAS::FflasRight, FFLAS::FflasNoTrans, s,
+                                        0, N - s* (brow + 2) - 1, H + s * N * (brow + 1) + s * (brow + 2), N,
+                                        q); // Apply permutation q to H
+                        FFLAS::fassign (Fi, s, s, H + s * N * (brow + 1) + s * (brow + 2), N,
+                                        V + ldv * s * (brow + 1), ldv); /* Sould not cause any trouble even if
+                                                                           last block*/
+                        // if (brow != k - 3) // Sould not cause segfault now
+                        // // shifts \hat H down
+                        // FFLAS::fassign (Fi, s, N - s * (brow + 3),
```

Comment:
if this fassign is no longer need, just remove it

Reply:
Fixed in [9dc4531]